client/src/components/DatePicker/DatePicker.stories.js

```diff
@@ -1,14 +1,14 @@
-import React from 'react';
-import { storiesOf } from '@storybook/react';
+// import React from 'react';
+// import { storiesOf } from '@storybook/react';
 
-import DatePicker from './DatePicker';
```

Comment:
Can you remove these files from your PR?

Reply:
all the files not related to your component

Reply:
fixed